resources/views/layout.blade.php

```diff
@@ -3,6 +3,7 @@
 <head>
     <title>Symposium for Speakers & Conferences</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/packages/octicons/octicons.css" rel="stylesheet">
```

Comment:
Is there any reason not to put this after `app.css`? if not, I'd prefer to put it after, so that the primary page styles load first, not this icon style.